widget/embedded/src/components/HeaderButtons/HomeButtons.tsx

```diff
@@ -27,7 +27,10 @@ export function HomeButtons(props: HomeButtonsPropTypes) {
 
   return (
     <>
-      <Tooltip side="top" content={i18n.t('Refresh')}>
+      <Tooltip
+        container={document.getElementById('swap-box') as HTMLElement}
+        side="top"
+        content={i18n.t('Refresh')}>
```

Comment:
you can create a variable like tooltip container and remove duplicate code:
const tooltipContainer = document.getElementById('swap-box') as HTMLElement

Reply:
done